app/models/attribute.py

```diff
@@ -0,0 +1,27 @@
+from .. import db
+
+user_tag_associations = db.Table(
+    'users_association', db.Model.metadata,
```

Comment:
I was actually talking about changing the name of the table in the string `'users_association'` to `'user_tag_associations'`